src/parser/user-extractor-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import configuration from "../configuration/config-reader";
+import { GetActivity } from "../get-activity";
+import { GitHubIssue, GitHubIssueComment } from "../github-types";
+import { Result, Transformer } from "./processor";
+
+/**
+ * Creates entries for each bounty hunter with its associated comments.
+ */
+export class UserExtractorTransformer implements Transformer {
+  private readonly _configuration = configuration["user-extractor"];
+
+  get enabled(): boolean {
+    return true;
+  }
+
+  /**
+   * Checks if the comment is made by a human user, and if it not a command.
+   * @param comment
+   */
+  _checkEntryValidity(comment: GitHubIssueComment) {
+    return !!comment.user && !!comment.body && comment.user.type === "User";
+  }
+
+  _extractBountyPrice(issue: GitHubIssue) {
+    if (this._configuration["redeem-bounty"] === false) {
+      return 0;
+    }
+    const sortedPriceLabels = issue.labels
+      .reduce((acc, label) => {
+        const labelName = typeof label === "string" ? label : label.name;
+        if (labelName?.startsWith("Price: ")) {
+          const price = parseFloat(labelName.replace("Price: ", ""));
+          if (!isNaN(price)) {
+            acc.push(price);
+          }
+        }
+        return acc;
+      }, [] as Array<number>)
```

Comment:
```suggestion
      }, [] as number[])
```

Reply:
Any reason to prefer this syntax? I always use `Array` to avoid ambiguities such as `number | string[]` which is not `(number | string)[]`, `Array<number | string>` is never ambiguous, `Array<number | string[]>`

Reply:
I'm not sure I think I was trained by some linters at some point.

Reply:
> In TypeScript, both `Array<number>` and `number[]` declare an array of numbers. The choice between them is mostly stylistic:
> 
> - `number[]` is more concise and is commonly used for simple arrays.
> - `Array<number>` can be easier to read when using complex types, like arrays of functions or arrays of arrays.
> 
> Use the one that fits your coding style or project's coding guidelines.
https://chat.openai.com/share/3206263b-ba8d-4c4c-9890-5264fd0bdb02

Reply:
Maybe `Array<>` syntax is better.

Reply:
Force of habit for me too after having had too many brainfarts with arrays of arrays haha, will change it.